test/ring_bytecode_aevm_SUITE.erl

```diff
@@ -28,15 +32,15 @@ execute_identy_fun_from_ring_file(_Cfg) ->
     ok = aeb_disassemble:pp(Code),
 
     %% Create the call data
-    Call = {<<"main">>, 42},
-    {0, Data} = aer_data:to_binary(Call),
-    Res =
+    CallData = aer_abi:create_calldata(Code, "main", 42),
+    io:format("CallData: ~p~n", [CallData]),
```

Comment:
do you want to leave it over here?

Reply:
Ill remove in my next branch/PR when it actually will be used.